e2e/pages/portforward-page.ts

```diff
@@ -0,0 +1,12 @@
+import { Page, Locator } from 'playwright';
+import { expect } from '@playwright/test';
+
+export class PortForwardPage {
+    readonly page: Page;
+    readonly portForwardingContent: Locator;
```

Comment:
```suggestion
    readonly content: Locator;
```
Since this already the Port Forwarding page.  Same for `wslDescription` in `wsl-page.ts`.